lib/project_copier.rb

```diff
@@ -10,7 +10,8 @@ class ProjectCopier
     :tagged_resources,
     :avatar,
     :background,
-    { active_workflows: %i[tutorials attached_images] }
+    { active_workflows: %i[tutorials attached_images] },
+    { field_guides: %i[attached_images] }
```

Comment:
question on this: 
https://github.com/zooniverse/panoptes/pull/4258/files#diff-36605a4c75522dc2019f50020273c1e6b97ae1472374d4877b39eb81526c1d7fR7 (L7) pulls in a shallow copy of field_guides. 

My understanding is that this format `{ field_guides: %i[attached_images] }` is a deeper includes (I'm taking that as it includes a shallow copy of field_guides + anything deeper you tell it to include.) 

Am I off base in thinking that if we can remove L7 since we have L14 [this new line]?

Reply:
Yeah, this is redundant, good catch.